src/core.js

```diff
@@ -24,17 +24,24 @@ $.extend($.fn, {
 
 		if ( validator.settings.onsubmit ) {
 
-			this.validateDelegate( ":submit", "click", function( event ) {
+			this.validateDelegate( ":submit, :submit *", "click", function( event ) {
 				if ( validator.settings.submitHandler ) {
 					validator.submitButton = event.target;
 				}
+
+				// locate the actual "action" element
+				var actionTarget = $(event.target);
```

Comment:
couldn't we just check wheter `event.target` is of type input, and if not, `find` for the input starting from `event.target`?